_integration-schemas/pinterest-ads/v1/advertisers.md

```diff
@@ -0,0 +1,136 @@
+---
+tap: "pinterest-ads"
+version: "1"
+key: ""
+
+name: "advertisers"
+doc-link: "https://developers.pinterest.com/docs/redoc/adsreporting/#tag/advertisers"
+singer-schema: "https://github.com/singer-io/tap-pinterest-ads/blob/master/tap_pinterest_ads/schemas/advertisers.json"
```

Comment:
We may need to comment this out, as the repo may not be public at launch due to the API being private.